src/renderers/WebGLRenderer.js

```diff
@@ -1269,6 +1273,11 @@ class WebGLRenderer {
 
 			const currentRenderTarget = _this.getRenderTarget();
 			_this.setRenderTarget( _transmissionRenderTarget );
+
+			_this.getClearColor( _currentClearColor );
+			_currentClearAlpha = _this.getClearAlpha();
+			if ( _currentClearAlpha < 1 ) _this.setClearColor( 0xffffff, 0.8 );
```

Comment:
@WestLangley the only thing I changed about your PR when I pulled it in was to set this clear color from gray to white. I was also able to remove the strange mix I'd added to the fragment shader, so I definitely prefer your approach. It's also yielding better renders.

Reply:
I do think my PR (updated in #26003), which uses the original gray color `0x7f7f7f`, is preferable. In my testing of a variety of models with only a CSS background, white is too bright.

Reply:
You're probably right - let me check a few more examples.